### DISCUSSION TITLE
Question/ Bug Regarding Sticker

### DISCUSSION THREAD

**Discussion by ChrisN944** [ByMykel/CSGO-API#122] [Q&A] [answered]
I am using CSGO-API/api/en/stickers.json to get all of the sticker data for a fun little project I am working on. However, I experienced an issue while trying to parse the JSON. The player "910" from "The MongolZ" has his "tournament_player" parameter saved as 910 (the integer) as opposed to "910" (the string). I am curious why this was done this way, as all other players with stickers have their "tournament_player" parameter saved as a string.

**Comment by ByMykel** [comment, marked as answer]
Hi, seems like it's a problem with the parser I am using.

Checking [items_game.txt](https://raw.githubusercontent.com/csfloat/cs-files/master/static/items_game.txt):
![image](https://github.com/user-attachments/assets/f12bc61b-1303-4e03-950c-07fe103b5e39)

Checking the [parsed JSON](https://raw.githubusercontent.com/ByMykel/counter-strike-file-tracker/main/static/items_game.json):
![image](https://github.com/user-attachments/assets/bc2b43b7-f439-4304-affd-50e7344d44ef)

I think i will convert every name to string to avoid this error.

**Reply by ChrisN944** [reply]
The obvious issue here is that someone decided to name themselves 910. Thanks for addressing the issue so quickly.

**Reply by ChrisN944** [reply]
Seems the parser was updated, but the JSON may need to be regenerated. Again, thank you for your work.

**Reply by ByMykel** [reply]
Yeah right now the script only runs 1 time every day and it checks that the manifestId has been updated. So i will remove the manifestId's and run it now.
